src/components/built-in-rules/CustomTabs.js

```diff
@@ -10,6 +10,7 @@ import Link from "@mui/material/Link";
 import { navItems, tabsItems } from "../../../static/builtInRulesNavbarItems";
 import { useHistory } from "@docusaurus/router";
 import { useColorMode } from "@docusaurus/theme-common";
+import { Typography } from "@mui/material";
 
 const handleTabChange = () => {
   const path = location.hash.split("#")[1];
```

Comment:
you got the same logic of `{location.hash.split("#")[1]`
you can abstruct it by creating a const with the value of `{location.hash.split("#")[1]`
and use it in both cases, here inside `handleTabChange` and also inside `Typography` `The following rules are enabled in policy`